Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-commander/user-guide/lisk-core.adoc

= Managing Lisk Core
Mona Bärenfänger <mona@lightcurve.io>

> :description: Technical reference about all available commands to manage Lisk Core with Lisk Commander.  For command it lists all available options, a usage example and an example response.

Reviewer: ```suggestion
:description: Technical reference regarding each available command used to manage Lisk Core with Lisk Commander.  For all commands all available options, a usage example and an example response are listed.
```
